.github/workflows/ci.yml

```diff
@@ -21,10 +25,35 @@ jobs:
           cache: yarn
       - name: Install Dependencies
         run: yarn install --frozen-lockfile
-      - name: Lint
+      - name: Lint Addon
+        run: yarn lint
+        working-directory: addon
+      - name: Lint Test App
         run: yarn lint
+        working-directory: test-app
       - name: Run Tests
         run: yarn test:ember
+        working-directory: test-app
+
+#  test-docs:
+#    name: "Test Docs"
+#    runs-on: ubuntu-latest
+#
+#    steps:
+#      - uses: actions/checkout@v2
+#      - name: Install Node
+#        uses: actions/setup-node@v2
+#        with:
+#          node-version: 12.x
+#          cache: yarn
+#      - name: Install Dependencies
+#        run: yarn install --frozen-lockfile
+#      - name: Lint
+#        run: yarn lint
+#        working-directory: docs
+#      - name: Run Tests
+#        run: yarn test:ember
+#        working-directory: docs
```

Comment:
Delete?

Reply:
@lukemelia deleted. This will be needed for part 2 when docs app will be extracted into separate app on `/docs` folder